cmd/preflight/cmd/check_container.go

```diff
@@ -89,6 +90,9 @@ func checkContainerCmd(runpreflight runPreflight) *cobra.Command {
 	flags.String("platform", rt.GOARCH, "Architecture of image to pull. Defaults to runtime platform.")
 	_ = viper.BindPFlag("platform", flags.Lookup("platform"))
 
+	_ = viper.BindEnv("cpuprofile")
```

Comment:
No flags? Just environment vars?

Reply:
Correct. It's not really something the end user is going to use. This is a dev tool IMHO.

Reply:
I don't feel strongly one way or the other on this, but if others disagree and we need a flag, we could always go the hidden flag route like we have with some other flags.